server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -144,6 +147,7 @@ fn create_filtered_query<'a>(
 ) -> BoxedChangelogQuery {
     let mut query = changelog_deduped::dsl::changelog_deduped
         .filter(changelog_deduped::dsl::cursor.ge(earliest.try_into().unwrap_or(0)))
+        .filter(changelog_deduped::dsl::is_sync_update.eq(false))
```

Comment:
Hmm, I think this would need to be exposed in ChangeLogFilter, and used similar to `get_active_records_on_site_filter`, this would also affect push all push test: https://github.com/openmsupply/open-msupply/compare/1293-Prevent-circular-name-sync...1293-example

Reply:
[0e21710](https://github.com/openmsupply/open-msupply/pull/1338/commits/0e217104c54f6eb37b99fe5be5c6f54dfd86c43a) thanks Andrei~